src/tests/JIT/HardwareIntrinsics/Arm/Shared/LoadVectorx2Test.template

```diff
@@ -20,7 +20,6 @@ namespace JIT.HardwareIntrinsics.Arm
     public static partial class Program
     {
         [Fact]
-        [ActiveIssue("https://github.com/dotnet/runtime/pull/92855#issuecomment-1746078670", TestRuntimes.Mono)]
```

Comment:
just to make sure, have you verified these tests ran and passed on mono?

Reply:
Added comment below to show local test run result.